src/bundles/pix_n_flix/types.ts

```diff
@@ -1,26 +1,36 @@
 export type VideoElement = HTMLVideoElement & { srcObject?: MediaStream };
+export type ImageElement = HTMLImageElement;
 export type CanvasElement = HTMLCanvasElement;
 export type ErrorLogger = (
   error: string | string[],
   isSlangError?: boolean
 ) => void;
 export type TabsPackage = {
-  onClickStill?: () => void;
-  handleSwapModes?: () => void;
+  onClickStill: () => void;
+};
+export type InputFeed = 'camera' | 'imageURL' | 'videoURL' | 'local';

```

Comment:
Although TypeScript helps to ensure type safety such that it is these strings that can be assigned to variables of type `InputFeed`, it may be beneficial to instead use an exported enum, with its members defined once in a single location. They could then be referenced in all other places and used for comparisons, eg `inputFeed === InputFeed.IMAGE_URL`. This could allow for easier refactoring in the future without the raw string values being typed out in many places, and it may serve to eliminate potential bugs in the absence of static analysis (eg `inputFeed === 'imageUrl'` would never be true)

Reply:
We have changed it to an enum. We will update the code soon.